cmd/ddev/cmd/config_test.go

```diff
@@ -375,6 +375,51 @@ func TestConfigSetValues(t *testing.T) {
 	assert.Equal("FOO=bar", app.WebEnvironment[1])
 	assert.Equal("SPACES=with spaces", app.WebEnvironment[3])
 	assert.Equal(webEnv, app.WebEnvironment[2])
+
+	// test docroot locations
+	testMatrix := []struct {
+		description string
+		input       string
+		expected    string
+		error       string
+	}{
+		{"empty docroot", "", "", ""},
+		{"dot docroot", ".", "", ""},
+		{"fail for outside approot", "..", "", "is outside the project root"},
```

Comment:
Maybe I don't understand, but it looks like this is looking for the literal "..", but we probably want to be testing for something that *begins with* ".."? No need to make it fancier, it's already a good test as is.